netbox/netbox/views/generic/mixins.py

```diff
@@ -9,25 +9,58 @@
 
 
 class ActionsMixin:
-    actions = ('add', 'import', 'export', 'bulk_edit', 'bulk_delete')
-    action_perms = defaultdict(set, **{
+    # Map action names to the set of required permissions for each
```

Comment:
IMHO it might be useful here to have a doc-comment of standard actions - basically just bulk_rename and bulk_sync add here.  I think bulk_remove_devices is the only special case / used once.